Make every formatter optional (#15)
DanielNoord/pydocstringformatter: .github/CONTRIBUTING.md

- Implement a Formatter by inheriting from ``pydocstringformatter.formatting.Formatter``
- Add your new formatter to ``pydocstringformatter.formatting.FORMATTERS``

> - Create a clear docstring because this will be user-facing: it's what will be seen in the help.
> - Choose a proper name because this will be user-facing: the name will be used for option of the CLI.

Reviewer: ```suggestion
- Write a clear docstring because this will be user-facing: it's what will be seen in the help message for the formatters command line option.
- Choose a proper name because this will be user-facing: the name will be used to turn the checker on and off via the command line or config files.
```

- Implement a Formatter by inheriting from `pydocstringformatter.formatting.Formatter`
- Add your new formatter to `pydocstringformatter.formatting.FORMATTERS`

> - Write a clear docstring because this will be user-facing: it's what will be seen in

Reviewer: I just thought about this. What do we do with the help message if the docstring contains both a summary and body?
Not something to be fixed in this PR perhaps, but we might want to add a `TODO` and open an issue to track.

Author: See #28

>   the help message for the formatters command line option.

Reviewer: ```suggestion
  the help message for the formatter's command line option.
```

- Choose a proper name because this will be user-facing: the name will be used to turn

>   the checker on and off via the command line or config files.

Reviewer: ```suggestion
  the formatter on and off via the command line or config files.
```

Little `pylint` slipping in here? 😄

Author: Haha, yeah muscle memory... I fixed the function name just before 😄
